cypress/e2e/14-timeline/timeline.spec.ts

```diff
@@ -143,6 +143,37 @@ describe("Timeline", () => {
             });
         });
 
+        it("should set size and position to event toggle on expanded view source event", () => {
```

Comment:
I wonder if we can re-organise the tests to have one test that goes through all the event source bits. In this particular place I am thinking of merging it with https://github.com/matrix-org/matrix-react-sdk/blob/develop/cypress/e2e/14-timeline/timeline.spec.ts#L154

While it is nice to separate every case it may take too long in the future when we are adding more and more tests.

Reply:
Probably I just don't know what I am doing (ie. ignorance), but I saw this part of the docs: https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/cypress.md#good-test-hygiene

> 1. Test a well-isolated unit of functionality. The more specific, the easier it will be to tell what's wrong when they fail.
> 2. Don't depend on state from other tests: any given test should be able to run in isolation.

Is it fine to merge several tests into one, if the same function is tested?

Reply:
> Probably I just don't know what I am doing (ie. ignorance), but I saw this part of the docs: https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/cypress.md#good-test-hygiene

Good point. Sorry for confusing you. Going to discuss that part of the docs.

> Is it fine to merge several tests into one, if the same function is tested?

I would say yes as long as it tests the same bits and it shares the steps getting there. Here it is all about the event source view. On the other hand I would not mix up „sign up“ and „event source view“ tests for example.

Reply:
> I would say yes as long as it tests the same bits and it shares the steps getting there. Here it is all about the event source view. On the other hand I would not mix up „sign up“ and „event source view“ tests for example.

That totally makes sense. It would be perfect if the policy would be available as documentation.